server/src/assets/assets.service.ts

```diff
@@ -21,11 +22,12 @@ export class AssetsService {
   }
 
   async getAssets(getAssetsDto: GetAssetsDto): Promise<Asset[]> {
-    const { limit, offset, ...rest } = getAssetsDto;
+    const { limit, offset, title, ...rest } = getAssetsDto;
+    const whereParams = title ? { title: Like(`%${title}%`), ...rest } : { ...rest };
```

Comment:
Thanks for looking at this. Getting the search functionality to work is fantastic. 

One thing about `Like` is that it is only a temporary work around to doing more performant queries like full-text search.